src/buildifier/buildifier_availability.ts

```diff
@@ -33,9 +35,15 @@ const BUILDTOOLS_RELEASES_URL =
  */
 export async function checkBuildifierIsAvailable() {
   const buildifierExecutable = getDefaultBuildifierExecutablePath();
+
   // Check if the program exists (in case it's an actual executable and not
   // an target name starting with `@`).
-  if (!buildifierExecutable.startsWith("@")) {
+  const isTarget = buildifierExecutable.startsWith("@")
+
+  // Check if the program exists as a relative path of the workspace
+  const isRelative = fs.existsSync(path.join(vscode.workspace.workspaceFolders?.[0]?.uri?.fsPath, buildifierExecutable));
```

Comment:
Please use async APIs instead of the sync API. You should be able to use `util.promisify` to get a promise-based variant of `fs.exists` which you can then `await` on. See https://github.com/bazelbuild/vscode-bazel/blob/8972b0eb8270efe471245f5846efb335fd760fda/src/bazel/bazel_info.ts#L20 for an example